src/recap

```diff
@@ -233,28 +233,28 @@ print_sar() {
   sar "${FLAGS}" >> "${LOGFILE}"
 }
 
-# print the output of "apachectl fullstatus" to the specified file
-print_httpd_fullstatus() {
+# Print the output of the web server status page to the specified file
+print_http_fullstatus() {
   local LOGFILE="$1"
-  echo "Apache Status report" >> "${LOGFILE}"
-  apachectl fullstatus >> "${LOGFILE}"
+  echo "Web Status report" >> "${LOGFILE}"
+  curl -Ls "${OPTS_STATUSURL}" 2>/dev/null >> "${LOGFILE}"
```

Comment:
Should we make a new 'OPTS_CURL="-Ls"' here? I thought "what if the user has https://... and needs to use the -k (ignore cert mismatch) with curl?" Technically you could add this to OPTS_STATUSURL (="-k https://...") but that is sort of ugly.